cirrus/google_cloud/iam.py

```diff
@@ -196,13 +218,6 @@ def __init__(self, name):
         Args:
             name (str): The name of the Google role
         """
-        # If the name provided already starts with the prefix, remove it
-        if (
-            name.strip()[: len(GooglePolicyRole.ROLE_PREFIX)]
-            == GooglePolicyRole.ROLE_PREFIX
-        ):
-            name = name.strip()[len(GooglePolicyRole.ROLE_PREFIX) :]
```

Comment:
if you want to put this back (see my other comment) what do you think of this?
```
self.name = self.name.strip()
if self.name.startsWith(GooglePolicyRole.ROLE_PREFIX):
  self.name = self.name[len(GooglePolicyRole.ROLE_PREFIX):]
```

Reply:
I'll do something like this to maintain as much backwards compatibility as possible (and add a clarifying comment)